Makefile

```diff
@@ -1,3 +1,7 @@
+release:
+	docker build -t release -f Dockerfiles/Dockerfile.release .
+	docker run -e GITHUB_ACCESS_TOKEN='${GITHUB_ACCESS_TOKEN}' -e GITHUB_USER='${GITHUB_USER}' -e GITHUB_EMAIL='${GITHUB_EMAIL}' -e GITHUB_GPG_KEY_FILE='${GITHUB_GPG_KEY_FILE}' -v ${PWD}:/go/src/github.com/CMSgov/bcda-app release
```

Comment:
Consider adding the `--rm` flag here to clean up the docker container when we're finished.